Dockerfile

```diff
@@ -74,6 +75,24 @@ ARG NGX_BROTLI_COMMIT
 ARG HEADERS_MORE_VERSION
 ARG CONFIG
 
+ARG MAXMIND_VERSION=1.2.1
```

Comment:
https://github.com/maxmind/libmaxminddb/releases says that the latest version of this lib is 1.6.0. Can we try to use it here? Or are there some incompatibilities that force us to stick to v1.2.1?

Reply:
sure try it. i just have copied them from another repo

Reply:
@macbre changed it to 1.6.0